### DISCUSSION TITLE
Translate settings UI

### DISCUSSION THREAD

**Discussion by Res777** [rg35xx-cfw/Koriki#90] [Q&A]
I have to give the 35xx with koriki OS to my nephew who doesn't know English. I converted a theme into my language with pngs, but I can't find the strings for the system settings menu, neither from windows nor from commander, does anyone know in which path they are located?

**Comment by acmeplus** [comment]
Sorry for the delay answering, but unfortunately the strings for the settings cannot be changed without updating the code and recompiling. Simplemenu does not have that capability. I'm working on simplermenu+ that includes full translation among many other features, but it's not ready yet.
